go.mod

```diff
@@ -179,4 +179,6 @@ replace (
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
 )
 
+replace github.com/cosmos/gogoproto => ../gogoproto
```

Comment:
Same as https://github.com/cosmos/cosmos-sdk/pull/13838#discussion_r1020771943. 
Wait for cosmos/gogoproto tag.